Library/Homebrew/os/mac.rb

```diff
@@ -24,7 +24,9 @@ def locate tool
       # Give the name of the binary you look for as a string to this method
       # in order to get the full path back as a Pathname.
       (@locate ||= {}).fetch(tool) do |key|
-        @locate[key] = if File.executable?(path = "/usr/bin/#{tool}")
+        @locate[key] = if File.executable?(path = "#{Formula['cctools'].opt_prefix}/bin/#{tool}")
```

Comment:
I feel like we probably want to make the case more complex here: use the Xcode tools if present, and only fall back to a Homebrew cctools otherwise. We may want to push some logic into the places that consume this method, rather than putting it into `MacOS.locate`.